examples/armada-manifest-v1.yaml

```diff
@@ -0,0 +1,101 @@
+---
+schema: armada/Chart/v1
+metadata:
+  schema: metadata/Document/v1
+  name: ucp-dependecy
+data:
+  name: helm-toolkit
```

Comment:
Consider using a more specific key here like `chartName` so that there isn't confusion between this and the `metadata.name` field.